### ISSUE TITLE
Mame layouts out of date

### ISSUE THREAD

**Issue by huwpascoe** [libretro/RetroArch#14548]
Mame has silently changed the layout spec and the video_layout module needs updating
https://docs.mamedev.org/techspecs/layout_files.html

I can make changes but would rather drop support for the original format (with the specific tags like &lt;background&gt; and &lt;overlay&gt;)
if everyone's fine with that?

related #13570

**Comment by hizzlekizzle** [comment]
I agree that a clean break is the best way to go. AFAIK, the artwork sources have all left the old format behind, so it's no big loss dropping the obsolete tags.

**Comment by LibretroAdmin** [comment]
Go ahead I guess.
